src/main/webapp/search.js

```diff
@@ -66,8 +66,9 @@ async function getSearchResultsHelper(document, window) {
 
 /** Fetches the list of books for the topic the user searched for. */
 async function getBooks(topic) {
-    //to be placed by Google Books API
-    await fetch("/books?topic="+topic).then(response => response.json()).then((results) => {
+    //TODO: create load more button so we can get next 40 results
+    await fetch("https://www.googleapis.com/books/v1/volumes?q=" + topic + "&maxResults=40&key=AIzaSyB1IWrd3mYWJsTWOqK7IYDrw9q_MOk1K9Y").then(response => response.json()).then((results) => {
```

Comment:
Could you theoretically then have maxResults be a really large number here?

Reply:
The Books API limits it to 40, so you can only get 40 results at a time.